geth/jail/internal/loop/loop.go

```diff
@@ -61,18 +61,13 @@ func NewWithBacklog(vm *vm.VM, backlog int) *Loop {
 		vm:        vm,
 		tasks:     make(map[int64]Task),
 		ready:     make(chan Task, backlog),
-		accepting: true,
+		closeChan: make(chan struct{}),
 	}
 }
 
 // close the loop so that it no longer accepts tasks.
 func (l *Loop) close() {
-	l.lock.Lock()
-	defer l.lock.Unlock()
-	if l.accepting {
-		l.accepting = false
-		close(l.ready)
-	}
+	close(l.closeChan)
```

Comment:
As I understand, if this method would be called > 1 time for some reason, it will panic.
Maybe we should wrap `close(l.closeChan)` to `sync.Once`?

Reply:
it is called only internally, but sure won't hurt